promscale/installation/kubernetes/helm.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Install Promscale on Kubernetes using Helm
+excerpt: Install Promscale on a Kubernetes cluster using Helm
+product: promscale
+keywords: [Kubernetes, analytics, Helm]
+tags: [install]
+related_pages: 
+  - /promscale/:currentVersion:/installation/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+
+# Install Promscale with Helm charts
+You can install Promscale using Helm charts and the Helm charts must be installed
+in this order:
+1. Install the TimescaleDB Helm chart
+1. Install the Promscale Helm chart
+
+## Before you begin
+1.  Understand the [resource requirements][resource-requirements] for your cluster.
+1.  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
+1.  Install [Helm 3][helm] in your local machine.
+1.  Install [kubectl][kubectl] in your local environment and
+    [connect to your cluster][connect-to-cluster].
+
+When you install TimescaleDB on Kubernetes using a Helm chart with the default
+`values.yaml` file, the user credentials are randomly generated during
+installation. Therefore, the `helm upgrade` command does not rotate the
+credentials, because changing the database credentials would break the database.
+Instead, it continues to use the credentials generated during `helm install`.
+
+Hence, create your own `.yaml` file to use parameters other than those specified
+in the default `values.yaml`. You can name this file `<MY_VALUES.yaml>`. For
```

Comment:
```suggestion
in the default `values.yaml`. Use the syntax `<MY_VALUES.yaml>` when you name the file. For
```

The original wording made it sound like you were saying the file *must* be named `<MY_VALUES.yaml>`, when it's really replaceable (judging by the format).